### PR TITLE
update default config with stress test values

### PR DESCRIPTION
**Pull request by delta1**
## Description

We've been using these config values for the stress tests, which means we probably want base nodes to use these by default

## Motivation and Context

Bigger buffers for base node and base node wallet, higher rate limits for the same buffers. Higher base node query timeout, tx broadcasting timeout, shorter tx chain monitoring timeout (@hansieodendaal is this right, 15 instead of 30?) , higher tx direct and broadcast send timeouts. 

## How Has This Been Tested?

Default values are updated if not overwritten from config.toml

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [X] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [X] I'm merging against the `development` branch.
* [X] I ran `cargo-fmt --all` before pushing.
* [X] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by hansieodendaal** [comment]
The defaults should not be changed in the source code, as these are minimum values that will work under normal circumstances. However, settings could be changed in the config toml files, but not as aggressive as used for the stress tests.

**Comment by delta1** [comment]
Fair enough - @mikethetike asked for this update so hopefully he has some feedback :)

**Comment by hansieodendaal** [comment]
The files to update would be `tari_config_sample.toml` and `windows.toml`, with those settings not commented out anymore.

**Comment by delta1** [comment]
Can do, just want this discussion agreed upon before doing that

**Comment by hansieodendaal** [comment]
Maybe
```
```

**Comment by hansieodendaal** [comment]
Maybe these?
```
[common]

buffer_size_base_node = 1000
buffer_size_base_node_wallet = 25000
buffer_rate_limit_base_node = 100
buffer_rate_limit_base_node_wallet = 100

[wallet]

base_node_query_timeout = 300
transaction_broadcast_monitoring_timeout = 300
transaction_chain_monitoring_timeout = 30
transaction_direct_send_timeout = 300
transaction_broadcast_send_timeout = 300
```

File: common/src/configuration/utils.rs
Comment (stringhandler) [inline common/src/configuration/utils.rs:78]: This number seems oddly low

**Review by stringhandler — COMMENTED** [review]
Looking at it again, I think that the buffer sizes are fine as they are in this PR, but the timeouts should all be `300`, as @hansieodendaal suggested.

**Review by stringhandler — APPROVED** [review]
Approving now since these figures should not cause any problems as they are

**Comment by hansieodendaal** [comment]
Hi @mikethetike, I believe approving this PR was a mistake. See my previous comment:

> The defaults should not be changed in the source code, as these are minimum values that will work under normal circumstances. However, settings could be changed in the config toml files, but not as aggressive as used for the stress tests.

@philipr-za, your opinion, please?

**Comment by philipr-za** [comment]
> Hi @mikethetike, I believe approving this PR was a mistake. See my previous comment:
> 
> > The defaults should not be changed in the source code, as these are minimum values that will work under normal circumstances. However, settings could be changed in the config toml files, but not as aggressive as used for the stress tests.
> 
> @philipr-za, your opinion, please?

For normal operation these values for the wallet are too high IMO

```
base_node_query_timeout = 300
transaction_broadcast_monitoring_timeout = 300
transaction_chain_monitoring_timeout = 30
transaction_direct_send_timeout = 300
transaction_broadcast_send_timeout = 300
```
Makes sense in the stress test to try reduce the query load on the BN but under normal operation you are waiting for 5 minutes between retries?

**Comment by delta1** [comment]
Hey I'm happy to send another PR to update this, I just need a final call on the values

**Comment by delta1** [comment]
Are we going with these values in the source code, or just as config examples? 

```
[common]

buffer_size_base_node = 1000
buffer_size_base_node_wallet = 25000
buffer_rate_limit_base_node = 100
buffer_rate_limit_base_node_wallet = 100

[wallet]

base_node_query_timeout = 300
transaction_broadcast_monitoring_timeout = 300
transaction_chain_monitoring_timeout = 30
transaction_direct_send_timeout = 300
transaction_broadcast_send_timeout = 300
```

**Comment by hansieodendaal** [comment]
Those should just be the defaults in the config files (`tari_config_sample.toml` and `windows.toml`), not in the code. The source code should be reverted to the original values as it impacts the mobile wallet.

**Pull request merged**
